libraries/AP_RangeFinder/AP_RangeFinder_LeddarOne.cpp

```diff
@@ -50,13 +62,24 @@ bool AP_RangeFinder_LeddarOne::get_reading(uint16_t &reading_cm)
 {
     uint8_t number_detections;
     LeddarOne_Status leddarone_status;
+    uint8_t index = 0;
+    uint32_t nbytes;
 
     if (uart == nullptr) {
         return false;
     }
 
     switch (modbus_status) {
     case LEDDARONE_MODBUS_STATE_INIT:
+        // clear read buffer
```

Comment:
if you wrap the inside of this case with { } then you can declare uint8_t and uint32_t inside the case.